cypress/integration/admin/login/login.spec.js

```diff
@@ -17,4 +19,195 @@ context("Login Page", () => {
             cy.findByText(Cypress.env("DEFAULT_ADMIN_USER_USERNAME")).should("exist");
         });
     });
+
+    it("try to access admin pages as anonymous user", () => {
+        cy.visit("/page-builder/pages");
+        cy.findByText(/sign in/i).should("exist");
+        cy.findByText(/forgot password?/i).should("exist");
+
+        cy.visit("/form-builder/forms");
+        cy.findByText(/sign in/i).should("exist");
+        cy.findByText(/forgot password?/i).should("exist");
+
+        cy.visit("/cms/content-models");
+        cy.findByText(/sign in/i).should("exist");
+        cy.findByText(/forgot password?/i).should("exist");
+    });
+
+    it("try to access non-existing page as logged user", () => {
+        const newUserAccountEmail = uniqid("cypress_", "@z1fihlo8.mailosaur.net");
+
+        let fullAccessGroupUser;
```

Comment:
Nitpick: do we really need `fullAccessGroupUser` here? In the L56 we assign the received `user` to it, but we're not utilizing it anywhere except for the following couple of lines.

Unless I'm missing smt here, I'd say we can remove this variable (the same goes for the tests below, where we use the same approach).

Reply:
You are right. I removed fullAccessGroupUser from all the tests. Cheers!